packages/http/src/mocker/HttpMocker.ts

```diff
@@ -168,3 +168,5 @@ function computeBody(
   }
   return undefined;
 }
+
+export const mocker = new HttpMocker();
```

Comment:
Do we want a singleton here? Is there any weird state stuff which can happen with this class?

Reply:
All the other components were singletons as well — I've just unified the code. The mocker is stateless, so singleton does not change the game. Ultimately I want to remove this as a class — the mocker should be nothing more than a function. Baby steps!